index.js

```diff
@@ -81,12 +82,7 @@ module.exports = function prettyFactory (options) {
     const prettifiedMessage = prettifyMessage({ log, messageKey, colorizer, messageFormat, levelLabel })
 
     if (ignoreKeys) {
-      log = Object.keys(log)
-        .filter(key => !ignoreKeys.has(key))
-        .reduce((res, key) => {
-          res[key] = log[key]
-          return res
-        }, {})
+      log = filterLog(log, ignoreKeys)
```

Comment:
Can you avoid calling this if there are no ignoreKeys?

Reply:
Is the check at line 84 sufficient? Or do you prefer an explicit length check as well?

Reply:
Oh I missed it! Great!

Reply:
Okie, I shall leave this as it is. Thanks!